src/main/java/com/solr/clientwrapper/rest/SolrSchemaResource.java

```diff
@@ -41,50 +35,60 @@
 
 	public SolrSchemaResource(CreateSolrSchema createSolrSchema, DeleteSolarSchema deleteSolrSchema,
 			UpdateSolarSchema updateSolrSchema, GetSolarSchema getSolarSchema) {
-
 		this.createSolrSchema = createSolrSchema;
 		this.deleteSolrSchema = deleteSolrSchema;
 		this.updateSolrSchema = updateSolrSchema;
 		this.getSolarSchema = getSolarSchema;
-	}
+	}			
 
 	@PostMapping("/create")
 	@Operation(summary = "/create-schema", security = @SecurityRequirement(name = "bearerAuth"))
-	public ResponseEntity<SolrSchemaDTO> create(@RequestBody SolrSchemaDTO solrSchemaDTO) throws SolrServerException,
-			IOException, URISyntaxException, ParserConfigurationException, InterruptedException {
+	public ResponseEntity<SolrSchemaDTO> create(
+			@RequestBody SolrSchemaDTO newSolrSchemaDTO) {
 		log.debug("Solr Schema Create");
-		SolrSchemaDTO solrSchemaDTO2 = createSolrSchema.create(solrSchemaDTO.getTableName(), solrSchemaDTO.getName(),
-				solrSchemaDTO.getAttributes());
-		return ResponseEntity.status(HttpStatus.OK).body(solrSchemaDTO2);
-
+		log.debug("(((((((((((()))))))) Received Schema as in Request Body: {}", newSolrSchemaDTO);
+		SolrSchemaDTO solrSchemaDTO = createSolrSchema.create(
+				newSolrSchemaDTO.getTableName(), 
+				newSolrSchemaDTO.getName(),
+				newSolrSchemaDTO);
+		return ResponseEntity.status(HttpStatus.OK).body(solrSchemaDTO);
 	}
 
-	@DeleteMapping("/delete/{name}")
+	@DeleteMapping("/delete/{tableName}/{name}")
 	@Operation(summary = "/delete-schema", security = @SecurityRequirement(name = "bearerAuth"))
```

Comment:
Delete method should take data from request body and not path parameter

Reply:
Ok, will talk about it with Mangesh. Web layer is implemented by him.